.github/workflows/scorecard.yml

```diff
@@ -14,8 +14,10 @@ on:
   push:
     branches: [ "develop", "main" ]
 
-# Declare default permissions as read only.
-permissions: read-all
+
+permissions:
+  contents: read
+  actions: read
```

Comment:
Same as above, could you please check if we need `actions: read` or it is safe to drop it?